apps/website/src/pages/docs/stepper.mdx

```diff
@@ -9,19 +9,46 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-stepper' />
-
 <LiveExample src='Stepper.main.tsx'>
   <AllExamples.StepperMainExample client:load />
 </LiveExample>
 
 In defined, multi-step user interactions, it is useful to inform the user of the number of steps involved in the process, and the current step within that sequence. A UI 'Stepper' guides the user through the steps and provides some interaction to return to previous steps. A standard example of a Stepper would be an e-Commerce checkout process, but there are many others where the user must perform sequential steps.
 
+## Variants
+
+There are two different styles of stepper, depending on the amount of steps required to complete the process.
+
+### Short Stepper
+
+The default "short" stepper example is suggested for processes with four (4) steps or less. Each step is named under the circle. This stepper is suggested if the step names can be summarized with very short labels. For example, instead of “Complete Billing Information”, simply use “Billing.”
```

Comment:
Move usage guidelines to Usage section. You could also add example layout there.

Reply:
The usage guidelines have been moved to the usage section

Reply:
The layout example has been updated. I decided to go with a simpler layout than the screenshots provided, but there's a small bug I'm running into with inputs that I'm tackling

![StepperLayoutExampleBug](https://user-images.githubusercontent.com/78564221/228389207-fb1dc854-2844-4745-b3a3-a279ceab79c5.gif)

Reply:
there is no need for state. you can fix the bug by using a different `key` for the different inputs